order_guidelines.md

```diff
@@ -1,36 +1,36 @@
 
-It is possible to put the first Cluster of the Segment in the Meta Seek to make it easier to find the actual content of the file.
-Meta Seek should also contain some padding (preferably outside and just after the Meta Seek) for a link to Tags, Chapters and Attachments, when they are not already found in the Segment. More padding may also be a good idea to add links in the future for other sections that may be appended in the Matroska file.
+It is possible to put the first `Cluster` of the `Segment` in the `SeekHead` Element to make it easier to find the actual content of the file.
+The `SeekHead` Element should also contain some padding (preferably outside and just after the `SeekHead` Element) for a link to `Tags`, `Chapters` and `Attachments` Elements, when they are not already found in the `Segment` Element. More padding may also be a good idea to add links in the future for other sections that may be appended in the Matroska file.
 
 ## Cues (index)
 
-It is much easier to put the Cues at the "end" of the segment, once all the Clusters have been written, otherwise it's hard to predict beforehand the place to reserve at the beginning of the segment. It is also not a big deal if Cues are at the end, given when the user wants to seek in a Matroska stream, it is going to jump somewhere, so it can seek to the Cues entry, read it and then seek to the best position according to the Cues entries. So the Cues <em>should</em> always be written after the Clusters. However the <a href="#cues_front">Cues could also appear at the front</a>. In this case the size of the Cues is usually very small compared to the video+audio bitrate of the stream. Even on small bitrate for a full length movie, the whole cue size represents only 0.01 to 0.1 second of download. Which is smaller than the time needed to seek on the network.
+It is much easier to put the `Cues` Element at the "end" of the `Segment`, once all the `Cluster` Elements have been written, otherwise it's hard to predict beforehand the place to reserve at the beginning of the `Segment` Element. It is also not a big deal if `Cues` are at the end of the `Segment` Element, given when the user wants to seek in a Matroska stream, it is going to jump somewhere, so it can seek to the `Cues` entry, read it and then seek to the best position according to the entries of the `Cues` Element. So the `Cues` Element <em>should</em> always be written after the `Cluster` Element. However the <a href="#cues_front">`Cues` could also appear at the front</a>. In this case the size of the `Cues` is usually very small compared to the video+audio bitrate of the stream. Even on small bitrate for a full length movie, the whole `Cues` Element size represents only 0.01 to 0.1 second of download. Which is smaller than the time needed to seek on the network.
```

Comment:
I don't think that the word `"end"` should be put in parenthesis here. I don't even know why that was the case in the original version.